Plugins/NativeServerPlugin/src/multi_peer_conductor.cpp

```diff
@@ -59,9 +59,17 @@ void MultiPeerConductor::SetDataChannelMessageHandler(const function<void(int, c
 
 void MultiPeerConductor::OnIceConnectionChange(int peer_id, PeerConnectionInterface::IceConnectionState new_state)
 {
+	// if we already know what state you're in, and it hasn't changed, don't do anything
+	if (connected_peer_states_.find(peer_id) != connected_peer_states_.end() &&
```

Comment:
This condition prevents **cur_capacity_** from being calculated inaccurately right?

Reply:
correct